From 0196b73176e01a824392f247884544624f482bed Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 20 Aug 2018 13:54:57 +0300
Subject: [PATCH] Makefile: add hl-fw-loader to gaudi/dali-release

The hl-fw-loader is a part of the embedded package.
Add it to the versioning system.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 655fe12..f3adefe 100644
--- a/Makefile
+++ b/Makefile
@@ -62,7 +62,7 @@
 endif
 
 #TODO: add btl once it is separated from br-ext
-RELEASE_REPOS := buildroot-external preboot u-boot linux armcp tbot
+RELEASE_REPOS := buildroot-external preboot u-boot linux armcp tbot fw_loader
 RELEASE_TAG_POINTER := latest-embedded-release
 
 .PHONY: gaudi-release dali-release
